app/openExchangeRates.js

```diff
@@ -15,7 +15,9 @@ module.exports.URL = 'http://openexchangerates.org/api/latest.json?app_id=';
 
 module.exports.init = function (fetchInterval) {
   fetchInterval = fetchInterval || 3600000;
-  intervalId = setInterval(module.exports.fetchOptions, fetchInterval, {});
+  const args = { live: true };
```

Comment:
We should probably make the interface consistent - let's add `options` as a second parameter  rather than just hard code this. Can you change this to allow passing in options as an arg but defaulting to live: true ?

Reply:
Hi. We could add an option `polling` defaulting to true. If the option is false, we could skip the `init` call to not have a setInterval call that is useless.

Reply:
I updated the implementation to be able to disable the polling by putting `live: false` to the config param.